StRoot/StGeant4Maker/etc/setup

```diff
@@ -0,0 +1,25 @@
+setup 64b
+
+module use /cvmfs/star.sdcc.bnl.gov/star-spack/spack/share/spack/modules/linux-rhel7-x86_64
+#module load root-6.24.00
+source /star/simu/simu/jwebb/ROOT/local/bin/thisroot.csh
```

Comment:
Can you use the the ROOT installed by Spack instead? This should suffice:

```
module load star-env-X.Y.Z-root-6.V.W
```

...and you don't need to set the environment variables by hand as you do later in this file.